pm/ticket.go

```diff
@@ -220,7 +220,11 @@ func (e *TicketExpirationParams) AuxData() []byte {
 }
 
 func ticketEV(faceValue *big.Int, winProb *big.Int) *big.Rat {
-	return new(big.Rat).Mul(new(big.Rat).SetInt(faceValue), new(big.Rat).SetFrac(winProb, maxWinProb))
+	return new(big.Rat).Mul(
+		new(big.Rat).SetFrac(faceValue, EVMultiplier),
```

Comment:
I don't think dividing by `EVMultiplier` is needed here. The formula for the ticket EV is:

```
ticketEV = faceValue * (winProb / maxWinProb)
```

This change would modify the formula to:

```
ticketEV = (faceValue / EVMultiplier) * (winProb / maxWinProb)
```